zerver/lib/actions.py

```diff
@@ -3813,7 +3815,7 @@ def do_create_realm(string_id: str, name: str,
     realm.save(update_fields=['notifications_stream', 'signup_notifications_stream'])
 
     if settings.BILLING_ENABLED:
-        do_change_plan_type(realm, Realm.LIMITED)
+        do_change_plan_type(realm, Realm.LIMITED, acting_user=None)
```

Comment:
You don't need to pass None here since the default value of acting_user is None. Please make the same change in other commits as well if required.